### PR TITLE
new module: Global Accelerator

### PR DESCRIPTION
**Pull request by swiknaba**
```terraform
module "global-accelerator" {
  source = "github.com/dbl-works/terraform//global-accelerator?ref=v2021.07.12"
  project            = "my-project"
  environment        = "production"
  load_balancers = [
    {
      region   = "eu-central-1"
      endpoint = "arn:aws:elasticloadbalancing:eu-central-1:${account_id}:loadbalancer/app/${project}-${environment}/xxx"
    },
    {
      region   = "us-east-1"
      endpoint = "arn:aws:elasticloadbalancing:us-east-1:${account_id}:loadbalancer/app/${ecs_name}/xxx"
    }
  ]
  # optional
  health_check_path = "/livez"
  health_check_port = 3000
  client_affinity   = "SOURCE_IP"
}
```

Tested on a multi-region deployed application -> works ✅

### CODE REVIEW

**Review by samkahchiin — APPROVED** [review]
LGTM 👍

File: global-accelerator/output.tf [hunk 1]
```diff
@@ -0,0 +1,7 @@
+output "ip_sets" {
+  value = ["${aws_globalaccelerator_accelerator.main-accelerator.*.ip_sets}"]
+}
+
+output "dns_name" {
+  value = ["${aws_globalaccelerator_accelerator.main-accelerator.*.dns_name}"]
```
[the remark this answers is not in the document]
Comment (swiknaba) [reply global-accelerator/output.tf:6]: It outputs only 1 endpoint

[the remark this answers is not in the document]
Comment (samkahchiin) [reply global-accelerator/output.tf:6]: Can just use `aws_globalaccelerator_accelerator.main-accelerator.dns_name` if there is only 1 endpoint?

[the remark this answers is not in the document]
Comment (swiknaba) [reply global-accelerator/output.tf]: good catch, thanks! adjusted that

[the remark this answers is not in the document]
Comment (swiknaba) [reply global-accelerator/output.tf]: image

**Pull request merged**